pkg/remotecmd/remotecmd.go

```diff
@@ -8,96 +8,86 @@ import (
 
 	"k8s.io/klog"
 
+	"github.com/redhat-developer/odo/pkg/kclient"
 	"github.com/redhat-developer/odo/pkg/log"
 )
 
-// ExecClient  is a wrapper around ExecCMDInContainer which executes a command in a specific container of a pod.
-type ExecClient interface {
-	ExecCMDInContainer(ComponentInfo, []string, io.Writer, io.Writer, io.Reader, bool) error
-}
-
 // ExecuteCommand executes the given command in the pod's container
-func ExecuteCommand(client ExecClient, compInfo ComponentInfo, command []string, show bool, consoleOutputStdout *io.PipeWriter, consoleOutputStderr *io.PipeWriter) (err error) {
+func ExecuteCommand(
+	client kclient.ClientInterface,
+	containerName string,
+	podName string,
+	command []string,
+	show bool,
+	consoleOutputStdout *io.PipeWriter,
+	consoleOutputStderr *io.PipeWriter,
+) (stdout []string, stderr []string, err error) {
```

Comment:
I see two places where `ExecuteCommand` is called. Both places call it like:
```go
_, _, err := ExecuteCommand(...)
```
I wonder if the original signature made sense as far as the return arguments are concerned. WDYT?